storybook-static/iframe.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><base target="_parent"><style>:not(.sb-show-main) > .sb-main,
```

Comment:
You accidentally committed all of the generated `storybook-static/` files. I've been meaning to add this directory to the .gitignore, and this reminded me, so I'm going to go ahead and add that now in a separate PR.

Reply:
ok

Reply:
I lied in my original comment and didn't submit a PR until just now: https://github.com/ShelterTechSF/sheltertech.org/pull/160

But please make sure you delete these files from your branch, even if you merge/rebase the `main` branch after that PR merges, since the `.gitignore` changes won't retroactively remove things that have been committed.

Reply:
ok